### DISCUSSION TITLE
Is Notepadqq "abandoned"?

### DISCUSSION THREAD

**Discussion by JGMelon22** [notepadqq/notepadqq#1007] [General]
Don't get me wrong, I really love the program and, besides the last release was back in 2018 and it still really stable, I'd love to know if there are any scheduled update to come?

**Comment by danieleds** [comment]
Hey @JGMelon22, thanks. 
As you can imagine, I have not much time left to follow the development (unfortunately...). I will gladly review and accept pull requests for bug fixes though, or new features if they are thought out beforehand.

**Comment by ericjs** [comment]
It seems there has been a flurry of activity lately! Is notepaddqq making a come back?

**Comment by MojtabaKD** [comment]
It depends on us all!
